### PR TITLE
Add shouldIncludeStyleguide option

### PR DESCRIPTION
**Pull request by jrowlingson**
#17 

Addresses part 1 of 2 (don't `app.import('etw.css')`)

~~It was unclear to me how to best approach a test for this particular change so I have opted to omit.~~

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,17 +29,19 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     let buildTarget = includer.options &&
-      includer.options['ember-cli-tailwind'] &&
-      includer.options['ember-cli-tailwind']['buildTarget'];
+      includer.options[this.name] &&
+      includer.options[this.name]['buildTarget'];
```
Comment (jrowlingson) [inline index.js]: Updating the reference to keep things consistent.

**Comment by samselikoff** [comment]
Awesome!

If you are feeling inclined, you could use [Mirage's node tests](https://github.com/samselikoff/ember-cli-mirage/tree/master/node-tests) as a reference for how to test the tree in each environment. (Mirage does something similar where it doesn't ship itself in `production`, unless explicitly enabled.)

If you don't think you'll have time, just let me know and we can punt on this for now.

Thanks!

**Comment by jrowlingson** [comment]
Thanks for the example! I have added a few tests to cover this option.

**Comment by samselikoff** [comment]
✨ Amazing ✨

**Pull request merged**